frontend/src/components/core/Sidebar/SidebarNav.test.tsx

```diff
@@ -112,86 +114,22 @@ describe("SidebarNav", () => {
     })
 
     it("are added to each link", () => {
-      const wrapper = shallow(<SidebarNav {...getProps()} />)
+      const buildAppPageURL = jest
+        .fn()
+        .mockImplementation(
+          (pageLinkBaseURL: string, page: IAppPage, pageIndex: number) => {
+            return `http://mock/app/page/${page.pageName}.${pageIndex}`
+          }
+        )
+      const props = getProps({ endpoints: mockEndpoints({ buildAppPageURL }) })
 
-      expect(
-        wrapper.find("StyledSidebarNavLink").map(node => node.props().href)
-      ).toEqual(["http://localhost/", "http://localhost/my_other_page"])
-    })
-
-    it("work with non-default port", () => {
-      window.location.port = "3000"
-      const wrapper = shallow(<SidebarNav {...getProps()} />)
-
-      expect(
-        wrapper.find("StyledSidebarNavLink").map(node => node.props().href)
-      ).toEqual([
-        "http://localhost:3000/",
-        "http://localhost:3000/my_other_page",
-      ])
-    })
```

Comment:
This, and other other removed test functions below, are now handled in `DefaultStreamlitEndpoints.test.ts`